Tests/TestingTests/BacktraceTests.swift

```diff
@@ -95,8 +95,42 @@ struct BacktraceTests {
       await runner.run()
     }
   }
+
+  @inline(never)
+  func throwNSError() throws {
+    let error = NSError(domain: "Oh no!", code: 123, userInfo: [:])
+    throw error
+  }
+
+  @inline(never)
+  func throwBacktracedRefCountedError() throws {
+    throw BacktracedRefCountedError()
+  }
+
+  @Test("Thrown NSError has a different backtrace than we generated", .enabled(if: Backtrace.isFoundationCaptureEnabled))
+  func foundationGeneratedNSError() throws {
```

Comment:
Guess it doesn't need `throws`?
```suggestion
  func foundationGeneratedNSError() {
```

Reply:
Yeah fair